meteor/lib/Settings.ts

```diff
@@ -49,7 +49,7 @@ export interface ISettings {
 	keyboardMapLayout: KeyboardLayouts.Names
 
 	// Name of the installation. Used to include custom implementations that differ from the main Fork. I.e. custom CSS etc. Leave undefined if no custom implementation is needed
```

Comment:
The comment still refers to installation name. I think it should say something along the lines of `CSS class applied to the body of the page. Used to include custom styling that differs from the main Fork. Leave undefined if no custom implementation is needed`

Reply:
Good catch. This is a great example of how fast comments becomes outdated and how easy it is to forget to update them :) 

The original idea of this setting was also to toggle features. But the new name doesn't really reflect that and since right now it only applies a CSS class, then I think your suggestion makes a lot of sense.